bot/cogs/reminders.py

```diff
@@ -104,7 +104,12 @@ def __init__(self, bot: Bot):
             name="It has arrived!"
         )
 
-        embed.description = f"Here's your reminder: `{reminder['content']}`"
+        if "jump_url" in reminder and reminder["jump_url"] != "invalid":  # keep backward compatibility
+            embed.description = (f"Here's your reminder: `{reminder['content']}`.\n"
+                                 f"[Jump back to when you created the reminder]({reminder['jump_url']})"
+            )
+        else:
+            embed.description = f"Here's your reminder: `{reminder['content']}`"
```

Comment:
I'd like this condition to be simplified, but it depends on the commends I've left for python-discord/site#288. Since `bot` will need to be merged before `site`¹, the best option for that would be:

```py
if reminder.get("jump_url"):
    # Do something if a jump_url is present
```
Using `.get` means we will get a default value (a falsy `None` in this case) when `"jump_url"` is not yet present in the response we get from `site`, an falsy empty string for historical reminders, and a truthy URL string for reminders set after this new feature.

I also think we should reformat this to a more DRY format, since you're currently having almost duplicate lines in your code that make it harder to maintain and to guarantee consistency in the front-end. (In fact, you've got a period after the reminder content in one path, but not in the other.)

We could tackle that like this:
```py
description = ["f"Here's your reminder: `{reminder['content']}`"]
if reminder.get("jump_url"):
    description.append(f"[Jump back to when you created the reminder]({reminder['jump_url']})")

embed.description = "\n".join(description)
```

Although, since we're only combining two strings here, a simple `+` concatenation would also do.

---
1) `bot` needs to be merged before `site` because `site` will require the `jump_url` field to be present after python-discord/site#288 is merged. Merging `site` before `bot` will therefore break the reminder feature.